### DISCUSSION TITLE
On compatibility

### DISCUSSION THREAD

**Discussion by Cherise-xu** [microsoft/AdaptiveCards#5546] [Q&A]
Hello, I found that IOS version 9 cannot be opened when using adaptive cards. Is there any solution.     use vue.js(2.0)

**Comment by jwoo-msft** [comment]
@Cherise-xu,
Are you using iOS AdaptiveCards SDK? If so, the AdaptiveCards is supported on iOS version 10 and up.

**Reply by Cherise-xu** [reply]
just using UIWebview

**Reply by jwoo-msft** [reply]
@Cherise-xu, I'm not following you. iOS AdaptiveCards SDK is supported in iOS version 10 and up. If your app is below the version, the iOS AdaptiveCards SDK may not work properly.
